cmd/oras/internal/display/status/interface.go

```diff
@@ -62,6 +62,8 @@ type CopyHandler interface {
 	PreCopy(ctx context.Context, desc ocispec.Descriptor) error
 	PostCopy(ctx context.Context, desc ocispec.Descriptor) error
 	OnMounted(ctx context.Context, desc ocispec.Descriptor) error
+	StartTracking(gt oras.GraphTarget) (oras.GraphTarget, error)
+	StopTracking()
```

Comment:
nit: For consistency with the existing code (in push/pull tty handlers), can we rename `StartTracking` to `TrackTarget`, remove `StopTracking` and use it in `cp.go` like this (from `pull.go`):

```
        dst, stopTrack, err := statusHandler.TrackTarget(dst)
	if err != nil {
		return ocispec.Descriptor{}, err
	}
	defer func() {
		_ = stopTrack()
	}()
```

Reply:
I would sooner update the push and pull handlers to this pattern because it does not require implementing this extra stopTrack function, the handler hides all that.

If that is okay, we can create an issue and assign it to me

Reply:
Done